### PR TITLE
LESQ-266 feat(homepage): homepage banner

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

create homepage banner
add banner to teachers homepage

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1949--oak-web-application.netlify.thenational.academy/teachers
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
A couple of changes:

- [ ] Can the content be full width so it aligns with the top nav
- [ ]  Padding all round set to 24px
- [ ] Mobile has a different view with button only
- [ ] Top green line should be thicker by 3 more pixels

File: src/pages/teachers/index.tsx [hunk 1]
```diff
@@ -43,51 +44,54 @@ const Teachers: NextPage<TeachersHomePageProps> = (props) => {
...
-            </Heading>
-            <Flex $flexDirection={"row"}>
-              <Typography $mr={16} $font="heading-7">
-                <OakLink page={"webinar-index"}>All webinars</OakLink>
-              </Typography>
-              <Typography $font="heading-7">
-                <OakLink page={"blog-index"}>All blogs</OakLink>
-              </Typography>
+    <>
+      <HomePageBanner />
```
Comment (njblore) [inline src/pages/teachers/index.tsx:48]: should this live in the `AppHeader`? seems a bit strange just floating above `AppLayout`

**Review by njblore — CHANGES_REQUESTED** [review]
Just left a question on the component structure, otherwise looks good

**Review by jack-skerrett-oak — APPROVED** [review]
I have nothing to add outside of Monika and Tash's points (see here that at least from an a11y standpoint all code should be within "accessibility landmarks" https://www.w3schools.com/accessibility/accessibility_landmarks.php)

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
Sorry one last small change. Looking at it I still think it's still quite tall. 

- [ ] set top and bottom padding to 16px so it's the same on mobile and desktop and all the way round

**Review by njblore — APPROVED** [review]

**Review by Monika-Koziol — APPROVED** [review]
Looking good

**Pull request merged**